### ISSUE TITLE
Duplicate Cities

### ISSUE THREAD

**Issue by sherred** [upcoming/upcoming-www#27]
Found one instance of duplicate cities, there appear to be two entries for Edinburgh, Scotland, United Kingdom when adding a city. Both can be added so Edinburgh appears twice in my city list. Unsure of there are other duplicate cities.

**Comment by citizenk** [comment]
Yes, there are other duplicates. For instance, Fribourg (Switzerland) appears at least twice:
- https://upcoming.org/place/fribourg-1643
- https://upcoming.org/place/fribourg-1665
And it's a rather small city at that... 

I couldn't point the cause, but if the list of cities is based on postal codes, in Switzerland at least there are cases were the same city can have different postal codes (but all in the same series): one for the PO Boxes, one for every quarter of the city, one generic for the whole city, etc.

Maybe later on we will be able to signal or correct such cases ourselves, but for now I guess we'll just have to live with it.

**Comment by luthian** [comment]
I added Boston, MA and Cambridge, MA and both came up duplicated at the top of the list.

**Comment by waxpancake** [comment]
Hmm, the placename db is from Who's on First at Mapzen. Let me look into it and see what I can find out.

**Comment by lostfocus** [comment]
I've looked into that DB before, mostly because the city nearby (Basel, Switzerland) is in there four times as well. It seems like they have problems with cities that might span and/or go right up to multiple counties.
